src/base/bootstrap-overrides/_base.scss

```diff
@@ -589,6 +589,24 @@ input[type=button], input[type=reset], input[type=submit] {
 	width: auto;
 }
 
+/*
+ * Explicit labels for checkbox and radio button inputs
+ *
+ * Allow the use of horizontal explicit labels instead of the default implicit
+ * labelling pattern for checkbox and radio button inputs provided by Bootstrap.
+ */
+.form-inline {
+	.label-inline {
+		font-weight: 400;
+		margin-bottom: 0;
+		padding-left: 1px;
+		vertical-align: middle;
+	}
+	.form-group + .form-group {
+		padding-left: 10px;
```

Comment:
The mrgn-lft-X class should be used instead.

Reply:
The class mrgn-lft-x adds margin not padding.